purchase_stock_picking_show_currency_rate/models/stock_move.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    price_unit_currency = fields.Float(
+        help="Techinal field to store the price unit of purchase order."
+    )
```

Comment:
Wouldn't it be better to extend _action_done() of stock.picking to keep the rate in a technical field (e.g., rate_at_done)? That way we don't need to touch stock.move and purchase.order.line models.